README.md

```diff
@@ -37,6 +37,7 @@
 
 - [ ] Try to parse the sources in a more effecient way to make the app faster overall: Try Serper API
 - [ ] Have the AI tool ignore video links like Youtube cause can't scrape them fast
+- [ ] Add a regenrate option for a user to re-generate
 - [ ] Make sure the answer correctly cites all the sources in the text & number the citations in the UI
 - [ ] Add sharability to allow folks to share answers
 - [ ] Automatically scroll when an answer is happening, especially for mobile
```

Comment:
### 😀 代码评分：85

#### ✅ 代码优点：
- 新增的“regenerate”选项为用户提供了额外的功能，增强了用户体验⛽。

#### 🤔 问题点：
- 新增的“regenerate”选项描述不够具体，可能会让用户困惑⚓。
- 列表项的格式不一致，影响可读性⛵。

#### 🎯 修改建议：
- 为“regenerate”选项提供更详细的描述，例如说明其功能和使用场景✈。
- 统一列表项的格式，使其更加一致和清晰⛴。

#### 💻 修改后的代码：
```markdown
- [ ] Try to parse the sources in a more efficient way to make the app faster overall: Try Serper API
- [ ] Have the AI tool ignore video links like Youtube because they can't be scraped fast
- [ ] Add a regenerate option for a user to re-generate the answer, explaining that this allows users to refresh the answer based on the same input
- [ ] Make sure the answer correctly cites all the sources in the text & number the citations in the UI
- [ ] Add sharability to allow folks to share answers
- [ ] Automatically scroll when an answer is happening, especially for mobile
```